src/main/java/com/nnaltakyan/core/auth/domain/verification/events/EventPublisher.java

```diff
@@ -0,0 +1,15 @@
+package com.nnaltakyan.core.auth.domain.verification.events;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Component;
+@Component
+public class EventPublisher {
+    @Autowired
+    private ApplicationEventPublisher eventPublisher;
+
+    public void publishEvent(String message){
```

Comment:
the publisher must always have as an argument event object and not a message, please change